doc/btp_gap.txt

```diff
@@ -637,6 +637,17 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+  Opcode 0x30 - Set RPA Timeout command/response
+
+    Controller Index: <controller id>
+    Command parameters: RPA Timeout (2 octets)
```

Comment:
Perhaps this should state the unit as well? Is it in seconds?

Reply:
It is in seconds, mentioned in description. I've checked other timeout parameters and noticed that they don't provide units, so omitted here to match style. However might be useful to have it.

Reply:
The description specifies the valid range, but it still doesn't explicitly say what the unit of the parameter is. The unit could be milliseconds and the valid range would still make sense. Point taken about this missing elsewhere, but I think it's better to be very clear about these details in a specification document.